dev.yml

```diff
@@ -6,4 +6,4 @@ up:
 commands:
   build:     cargo build
   test:      cargo test
-  run-built: target/debug/shadowenv ''
+  run-built: target/debug/shadowenv
```

Comment:
Not sure why there was a `''` before, but it was causing `dev run` to always fail

Reply:
ah, the `hook` subcommand used to be the default command, so that was the `$__shadowenv_data` parameter.